docs/server/source/introduction.md

```diff
@@ -15,8 +15,8 @@ Note that there are a few kinds of nodes:
 
 ## Setup Instructions for Various Cases
 
-* [Set up a local stand-alone BigchainDB node for learning and experimenting: Quickstart](quickstart.html)
-* [Set up and run a local dev/test node for developing and testing BigchainDB Server](dev-and-test/setup-run-node.html)
+* [Quickstart](quickstart.html)
```

Comment:
same HTTP 404 error here. may be relative links only work for sphinx and not on github?

Reply:
This works on the website and with sphinx. Might not work, on github.